src/core/client/stream/tabs/comments/helpers/getHTMLText.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * getHTMLText returns text representation of html.
+ * Includes a different implementation during test that works.
+ * @param html
+ */
+export default function getHTMLText(html: string) {
```

Comment:
To simplify this function, possibly replacing it with:

```ts
const getHTMLText = process.env.NODE_ENV === "test" ? () => {  /* */  } : () => {  /* */  };

export default getHTMLText;
```

Reply:
This is probably debatable, but I prefer my version for readability :-)

Reply:
I would be also worried if dead code elimination is smart enough to replace the unary operator?